source/MRViewer/MRMenu.cpp

```diff
@@ -813,37 +815,48 @@ void Menu::draw_helpers()
             AppendHistory( std::make_shared<ChangeNameAction>( "Rename object", obj ) );
             obj->setName( renameBuffer );
             show_rename_modal_ = false;
+            ImGui::CloseCurrentPopup();
         }
         ImGui::SameLine( 0, p );
         if ( ImGui::Button( "Cancel", ImVec2( ( w - p ) / 2.f, 0 ) ) || esc_pressed_ )
```

Comment:
pls change using of `esc_pressed_` and `enter_pressed_ ` to `ImGui::GetIO().KeysDownDuration[GLFW_KEY_ESC] == 0.0f` and `ImGui::GetIO().KeysDownDuration[GLFW_KEY_ENTER] == 0.0f` and remove  `esc_pressed_` and `enter_pressed_ ` if possible